src/plugins/intel_gpu/src/graph/graph_optimizer/prepare_buffer_fusing.cpp

```diff
@@ -353,7 +353,7 @@ void prepare_buffer_fusing::run(program& p) {
                 // do not optimize crop if paddings are not properly aligned
                 for (auto& usr : node.get_users()) {
                     auto usr_layout = usr->get_output_layout();
-                    if (usr_layout.format == format::b_fs_yx_fsv16 &&
+                    if (!usr->is_type<quantize>() && usr_layout.format == format::b_fs_yx_fsv16 &&
```

Comment:
Why do we need special treatment for quantize?

Reply:
quantize follows the format of input. But, at this time, passing prepare buffer fusing, quantize format is bfyx. Therefore, to ignore format checking, this condition is added.

Reply:
I removed this condition. it doesn't need anymore.